platforms/enterprise/enterprise/src/main/java/org/gradle/internal/enterprise/GradleEnterprisePluginBackgroundJobExecutors.java

```diff
@@ -42,6 +43,15 @@ public interface GradleEnterprisePluginBackgroundJobExecutors {
      */
     Executor getUserJobExecutor();
 
+    /**
+     * Run some code without tracking its inputs.
+     * <p>
+     * All code that runs during {@code supplier.get()} is exempt from configuration input tracking.
+     * That code can for example read files from disk or system properties at configuration time without
+     * them becoming a configuration input.
+     */
+    <T> T withConfigurationInputTrackingDisabled(Supplier<T> supplier);
```

Comment:
🤔 I'm not sure if this interface is an appropriate place for the method. It is not related to background jobs other than being exempt from CC too.

Reply:
> Yes, I agree. Should I create a new service for this? Anything else that would go in the new service?

Most likely a new service is needed. I would go with a scary name, like `Unsafe`, or something along these lines. I don't know what else should go there, but it would also depend on the name you choose.

Reply:
I created a `DevelocityPluginUnsafeConfigurationService` that hosts the new method.